mne/decoding/tests/test_time_gen.py

```diff
@@ -296,18 +337,18 @@ def test_generalization_across_time():
     y[len(y) // 2:] += 2
     ys = (y, y + 1000)
     # Univariate and multivariate prediction
-    svc = SVC(C=1, kernel='linear')
+    svc = SVC(C=1, kernel='linear', probability=True)
+    reg = KernelRidge()
 
-    class SVC_proba(SVC):
-        def predict(self, x):
-            probas = super(SVC_proba, self).predict_proba(x)
-            return probas[:, 0]
+    def scorer_proba(y_true, y_pred):
+        roc_auc_score(y_true, y_pred[:, 0])
```

Comment:
yes it's related to https://github.com/mne-tools/mne-python/issues/2874. 

This issue is created by the fact that we allow `predict_proba` which does not follow the sklearn API. This is still not in the stable version, so we have a margin to remove this back and change plan. 

I personally think it simplifies a lot of scripts. Most people I know use probabilistic outputs.